docs/iot/guides/update-control-api.md

```diff
@@ -0,0 +1,134 @@
+# Tizen Update Control Native API
+
+- [Prerequisites](#prerequisites)
+- [Connecting to Server](#connecting-to-server)
+- [Checking the Latest Version](#checking-the-latest-version)
+- [Downloading and Updating to New Version](#downloading-and-updating-to-new-version)
+- [Making an Update Reservation](#making-an-update-reservation)
+- [Update Properties](#update-properties)
+
+You can control the system software version of IoT device using update control API.
+The control of system software version includes:
+- Checking the latest version
+- Downloading and updating to the latest version
+- Making a reservation for update
+
+The update procedure and the server might be different according to the platform developer.
+Therefore, this API just offers general functions for update, and platform developers must develop the application using update control API properly.
+
+>  **Note**
+>
+> As a default, the update agent works with [SmartThings Device Manager](https://console.smartthingsdm.com/) is available.
+
+## Prerequisites
+
+To enable your application to use the update control functionality:
+
+1. To use the update control API (in application), the application has to request permission by adding the following privileges to the `tizen-manifest.xml` file:
+
+```xml
```

Comment:
Please add indentation to this code block.
As this code block is the sub item of above, it need 4 spaces in front of the line from 29 to 33.
Also please remove "xml".

Reply:
@mijin85cho It'll be applied.
But I have one question: is the code language of XML prohibited?

Reply:
@SumLee I rechecked the history with your question and found that `cpp` and `xml` are ok to use.
You can keep the language type. Sorry for the confusion.

Reply:
@mijin85cho That's OK. I'll restore the format then.